source/renderer/app/api/api.js

```diff
@@ -546,6 +546,24 @@ export default class AdaApi {
           { walletInitData },
           'random'
         );
+
+        if (legacyWallet) {
+          const { id } = legacyWallet;
+          const account = await this.getAddresses({
```

Comment:
@DeeJayElly we do not need this call here. It doesn't return the account anyways. It only returns a list of existing addresses which in this case would be empty.